docs/reference/project_package.md

```diff
@@ -0,0 +1,11 @@
+## Packaging data
+
+OC4IDS data must be published in a project package which acts an envelope for the data and contains important metadata.
+
+The project package schema describes the container document for publishing data using the OC4IDS project level schema.
```

Comment:
> The project package schema describes this envelope.

Merge into previous paragraph.

Reply:
Previous paragraph moved to the schema reference page in https://github.com/open-contracting/infrastructure/pull/89/commits/381b67bbdee4987956315a5b528d5b5f8538634e